ocs_ci/ocs/resources/namespacestore.py

```
@@ -319,6 +319,13 @@ def oc_create_namespacestore(
     nss_data["metadata"]["name"] = nss_name
     nss_data["metadata"]["namespace"] = config.ENV_DATA["cluster_namespace"]
 
+    if platform.lower() == constants.ARCHIVE_PLATFORM:
+        if not nss_tup or len(nss_tup) < 3 or not nss_tup[2]:
```

Ruff (0.16.1)

[warning] Use a single `if` statement instead of nested `if` statements

(SIM102)